docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.buttons.scss

```diff
@@ -41,33 +79,137 @@
   }
 
   @include hb-traditional {
-    font-family: $hb-traditional-font--sans;
+    &:active,
+    &:focus {
+      box-shadow: 0 0 0 hb-calculate-rems(1px) hb-get-pairing-color('secondary-active', $hb-traditional-default, $ht-traditional-pairings);
+      box-shadow: 0 0 0 hb-calculate-rems(1px) var(--palette--secondary-active);
+    }
+  }
+}
+
+@mixin hb-big-button {
+  @include hb-button;
+  font-size: hb-calculate-rems(18px);
+  border-width: hb-calculate-rems(3px);
+
+  @include hb-themes(('colorful', 'airy')) {
+    padding: hb-calculate-rems(8px) hb-calculate-rems(20px);
+  }
+
+  @include hb-traditional {
+    padding: hb-calculate-rems(15px) hb-calculate-rems(35px);
```

Comment:
This `looks` bigger than the design comps, but the numbers match so it's good!

Reply:
Should be fixed!